generators/generator-base.js

```diff
@@ -1515,7 +1515,14 @@ module.exports = class extends PrivateBase {
             this.log(chalk.cyan(`\nKeyStore '${keyStoreFile}' already exists. Leaving unchanged.\n`));
             done();
         } else {
-            shelljs.mkdir('-p', `${SERVER_MAIN_RES_DIR}config/tls`);
+            try {
+                shelljs.mkdir('-p', `${SERVER_MAIN_RES_DIR}config/tls`);
+            } catch (error) {
+                // noticed that on windows the shelljs.mkdir tends to sometimes fail
+                fs.mkdir(`${SERVER_MAIN_RES_DIR}config/tls/keystore.p12`, { recursive: true }, err => {
```

Comment:
Is `/keystore.p12` at the end of the path typo?

Reply:
@kaidohallik : Thanks for pointing it out. I've made a new pull request correcting this; https://github.com/jhipster/generator-jhipster/pull/11143